crates/ruff_linter/src/rules/airflow/rules/removal_in_3.rs

```diff
@@ -724,6 +740,9 @@ fn check_name(checker: &Checker, expr: &Expr, range: TextRange) {
                 Replacement::Name("airflow.sdk.get_parsing_context")
             }
 
+            // airflow.utils.db
+            ["create_session"] => Replacement::None,
+
```

Comment:
Do we need to add the `db` module in here? i.e., `["db", "create_session"]` based on the comment because we only match against `airflow.utils`, the remaining is matched here.

Reply:
Yes, just fixed. Thanks!